ledger-api/grpc-definitions/BUILD.bazel

```diff
@@ -135,13 +135,13 @@ proto_gen(
 genrule(
     name = 'docs',
     srcs = [':ledger-api-docs'],
-    tools = ["@pandoc_nix//:bin/pandoc", "post-process.sh"],
+    tools = ["post-process.sh"],
     outs = ['proto-docs.rst'],
     cmd = """
-        export PATH="$$( cd "$$(dirname "$(location @pandoc_nix//:bin/pandoc)")" ; pwd -P )":$$PATH
         unzip -q $(location :ledger-api-docs)
+        ls -la
         $(location post-process.sh)
-        mv docs_rst_mmd.rst '$@'
+        mv docs_rst_mmd.md '$@'
```

Comment:
Since this PR is already cleaning up the proto->md->rst mess, how about we de-confuse the situation and pass `docs.rst` to the proto-gen-docs plugin in [line 129 in this file](https://github.com/digital-asset/daml/pull/334/files#diff-35fe07ffebf49e0dc52e0c10ea03aec0R129) and change the name here and in `post-process.sh` as well? Or maybe even better, inline the 2 `sed` calls into this genrule directly. The script itself is not used anywhere else.

Reply:
Agreed! I'll have a go at that and see if I can manage it.